### PR TITLE
Bringing command line controls to toggle enviroments

### PR DESCRIPTION
**Pull request by ultrabitcjwhite**
Updating readme - we now have the following optional commands 

-s = Change Folder from Salt to user defined
example : node index.js -s=10K , will store in a s3 bucket '{bucket/folder}/avatars-g{genertation}-10K'
default : avatars-g{genertation}-{epoch}

-s = Replace Folder from avatars-g{genertation}-{epoch} to user defined
example : node index.js -s=10K , will store in a s3 url '{bucket/folder}/10K'

-e = Change environment folder
'dev' (default - urls are s3 bucket)
'release' (urls are CDN cached)
'static' (used for docs - urls are s3 bucket)

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -1,17 +1,69 @@
...
 
 const network = NETWORK.eth;
 
-// CDN root
-const CDN_ROOT='https://d66iprse6qzhg.cloudfront.net/';
+// S3/CDN roots based on enviroment
+const S3_ROOT_LIST = {
+  dev : 'https://bitverse-avatars.s3.amazonaws.com/',       // Intentionally using s3 so there is no caching issues
+  static : 'https://bitverse-avatars.s3.amazonaws.com/',    // Intentionally using s3 so there is no caching issues
+  prod : 'https://d3n9vm398ay3ts.cloudfront.net/',          // Production should always be behind a CDN
```
Comment (riniculous) [inline src/config.js:54]: We actually have a CDN setup?

Comment (ultrabitcjwhite) [reply src/config.js:54]: Yes , Infact we have 2 - one points to the root of the bucket and will be removed once we are on that new folder structure

**Review by riniculous — COMMENTED** [review]
@ultrabitcjwhite We have easy JS tools to write to S3 Files, to update saved data, level up avatars. etc etc. What are the implications to file writing, using a CDN?

**Comment by ultrabitcjwhite** [comment]
> 

Worst case, we would have on invalidate the cache , but the behavours are setup with json excluded from cache

**Pull request merged**